src/__tests__/App.test.jsx

```diff
@@ -0,0 +1,11 @@
+import React from 'react';
```

Comment:
@NikitaSedyx one more question related to project structure. 
I have seen that some people store `__test__` folder under root directory or store each test close to the class/component it test.
I understand that tests will not appear in result bundle because their are not used/imported by real components.

Reply:
you understand correctly, but what's a question?)